coalib/bearlib/abstractions/Linter.py

```diff
@@ -207,9 +207,9 @@ def get_executable():
             Returns the executable of this class.
 
             :return:
-                The executable name.
+                The full path to the executable file.
             """
-            return options['executable']
+            return shutil.which(options['executable'])
```

Comment:
sorry for the noob question, but why not just this https://github.com/coala/coala/pull/3534#discussion_r95806398 as you suggested instead going for cache?